apollo.sh

```diff
@@ -3,6 +3,7 @@ set -e
 
 TOP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
 source "${TOP_DIR}/scripts/apollo.bashrc"
+source "${TOP_DIR}/cyber/setup.bash"
```

Comment:
A question, I remember that this script has been sourced in `usr.bash_aliases`, why cannot CI find `${APOLLO_BAZEL_DIST_DIR}`

Reply:
> A question, I remember that this script has been sourced in `usr.bash_aliases`, why cannot CI find `${APOLLO_BAZEL_DIST_DIR}`？

Good question. @lx18233184051 Do you have any ideas, Big Xu?

Reply:
> > A question, I remember that this script has been sourced in `usr.bash_aliases`, why cannot CI find `${APOLLO_BAZEL_DIST_DIR}`？
> 
> Good question. @lx18233184051 Do you have any ideas, Big Xu?

I think the main reason is that CI runs apollo.sh directly without user.bash_aliases sourced.